README.md

```diff
@@ -64,6 +64,18 @@ $(function(){
   });
 ```
 
+### Auto Close after period of time
+```javascript   
+  $.jAlert({ //this is the normal usage
+    'title': 'Test',
+    'content': 'Howdy',
+    'theme': 'green',
+    'size': 'xsm'
+    'autoClose' : true,
+    'closeTime' : 3500
```

Comment:
I would prefer if you made this a single parameter with either false or a number in seconds.

Reply:
Great :) you can change like you want 

I think it's better with one parameter. This will be so helpful for just displaying a notification.

All change you like to do it's up to you !